test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml

```diff
@@ -47,17 +50,19 @@ spec:
       - name: root-mount
         hostPath:
           path: /
+      - name: cos-tools
+        hostPath:
+          path: /var/lib/cos-tools
+      - name: nvidia-config
+        hostPath:
+          path: /etc/nvidia
       initContainers:
-        # The COS GPU installer image version may be dependent on the version of COS being used.
-        # Refer to details about the installer in https://cos.googlesource.com/cos/tools/+/refs/heads/master/src/cmd/cos_gpu_installer/
-        # and the COS release notes (https://cloud.google.com/container-optimized-os/docs/release-notes) to determine version COS GPU installer for a given version of COS.
-
-        # Maps to gcr.io/cos-cloud/cos-gpu-installer:v2.1.10 - suitable for COS M109 as per https://cloud.google.com/container-optimized-os/docs/release-notes
-      - image: gcr.io/cos-cloud/cos-gpu-installer:v2.1.10
+      - image: "cos-nvidia-installer:fixed"
```

Comment:
is this image available in the cos images we're using?

Reply:
Yes, this container image is preloaded on the COS Node image.

Reply:
From the log. 
`Using image: cos-109-17800-147-22 from project: cos-cloud as node image`. it is actually using the base COS image without preloading. so we should update the exact version for 1.30 used here.

Reply:
@Dragoncell I see.

Reply:
From the release notes, v2.1.10 is the right one actually for image `cos-109-17800-147-22`

https://cloud.google.com/container-optimized-os/docs/release-notes/m109#cos-109-17800-147-9_